django_freeradius/static/django-freeradius/js/radcheck.js

```diff
@@ -2,7 +2,7 @@ django.jQuery(function ($) {
     'use strict';
     // stop here if in add mode
     if (!('.deletelink').length) { return; }
-    if (!gettext) { var gettext = function (text) { return text; }}
+    if (!gettext) { gettext = function (text) { return text; }; }
```

Comment:
Adding var creating two more messages saying `getext is out of scope`

Reply:
okey no problem